janusgraph-hbase/src/test/java/org/janusgraph/diskstorage/hbase/HBaseLogTest.java

```diff
@@ -14,26 +14,22 @@
 
 package org.janusgraph.diskstorage.hbase;
 
-import java.io.IOException;
-
+import org.janusgraph.HBaseContainer;
 import org.janusgraph.diskstorage.BackendException;
-
-import org.junit.jupiter.api.BeforeAll;
-
-import org.janusgraph.HBaseStorageSetup;
 import org.janusgraph.diskstorage.keycolumnvalue.KeyColumnValueStoreManager;
 import org.janusgraph.diskstorage.log.KCVSLogTest;
 
-public class HBaseLogTest extends KCVSLogTest {
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
 
-    @BeforeAll
-    public static void startHBase() throws IOException {
-        HBaseStorageSetup.startHBase();
-    }
+@Testcontainers
+public class HBaseLogTest extends KCVSLogTest {
+    @Container
+    public static final HBaseContainer _hBaseContainer = new HBaseContainer();
```

Comment:
Any reason for the underscore in naming? It seems you are not using underscore anywhere else (except this and HBaseMultiWriteStoreTest)

Reply:
I want to use underscore only if the variable is unused.